src/index.ts

```diff
@@ -4,14 +4,19 @@ import { parseIconSet } from "@iconify/utils"
 import { generateIconComponent, getIconCollections } from "./core"
 import { CollectionNames } from "../types"
 import { type Optional } from "./utils"
+import { IconsOptions } from "./types"
 
 export { getIconCollections, type CollectionNames }
 
 export const iconsPlugin = ({
   collections,
+  options,
 }: {
   collections: Record<string, Optional<IconifyJSONIconsData, "prefix">>
+  options?: IconsOptions
```

Comment:
can you expose `IconsOptions` at root level? ie `{collections: Record} & IconsOptions`

Reply:
It should be ok now